readme.md

```diff
@@ -415,6 +415,16 @@ You can use the `.todo` modifier when you're planning to write a test. Like skip
 test.todo('will think about writing this later');
 ```
 
+### Failing tests
+
+You can use the `.failing` modifier for submitting tests that are intended to be failures without killing CI process. And if the failing test does pass, it will fail, and it will remind you remove the failing modifier.
+
+```js
+test.failing('this test prove something wrong', t => {
```

Comment:
People should link to the relevant issue. We should model that behavior in the docs.

Also, a bit of language/style cleanup.

```js
// See: github.com/user/repo/issues/1234
test.failing('demonstrate some bug', t => {
  t.fail(); // test will count as passed
});
```